components/CarouselWithImages.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+const props = defineProps({
+  images: {
+    type: Array,
+    required: true
+  }
+})
+</script>
+
+<template>
+  <v-carousel v-if="props?.images && props.images?.length" 
+    width="100%"
+    height="445"
+    :show-arrows="false"
+    hide-delimiter-background
+    cycle
+    class="custom-carousel"
+  >
+    <template v-for=" (prop, index) in props.images" :key="index">
+      <v-carousel-item v-if="prop.src?.length && prop.alt?.length" 
+        :src="prop.src"
+        :alt="prop.alt"
+        cover
+        height="421"
+      >
+      </v-carousel-item>
+    </template>
+  </v-carousel>
+</template>
+
+<style>
```

Comment:
You should add  <style scoped> to avoid conflicts

Reply:
@PiotrDrechsler in fact, for some reason I can't add "scoped" here - my custom styles don't work on carousel in this case. That's why I added a "custom-carousel" class to component - to somehow narrow scope of this styles indeed.